Makefile

```diff
@@ -80,8 +80,8 @@ wave:
 	H2O_WAVE_PRIVATE_DIR="/download/@$(WORKDIR)/output/download" \
 	$(PIPENV) run wave run app
 
-.PHONY: wave-no-reload
-wave-no-reload:
+.PHONY: llmstudio
```

Comment:
Like the new name!  Should we remove or rename `make wave`, e.g. `make llmstudio-reload` ?

Reply:
feels natural :) 

actually, `make wave` is already in my muscle memory and quite quick to type, so I kind of like it for development. But I would also be fine with `make llmstudio-reload` if you have a strong preference.

Reply:
Yeah, I'm also used to it :D Thus, free to leave as is.